.travis.yml

```diff
@@ -44,30 +44,28 @@ matrix:
      language: python
      python: 3.5
      env: TESTENV=build_console
-     before_install: make deps
+     before_install:
+       - make deps
+       - pip install -r requirements_dev.txt
      script: make build_console
 
    - os: osx
      osx_image: Xcode 8.3.3
-     env: TESTENV=mac TOXENV=py27,py37
      language: generic
+     env: TESTENV=mac TOXENV=py27,gui35
```

Comment:
I think you should add also either py35, py37 or both here so that the cli tests would still get executed on mac for python 3, too.

Reply:
👍

Reply:
Reintroducing Python 3.7 will probably bring back the original CI problem, will let Travis bear this out and check on things in the morning.